presentation-api/index.html

```diff
@@ -0,0 +1,111 @@
+---
+feature_name: Presentation API
+chrome_version: 51
+feature_id: 6676265876586496
+---
+
+<h3>Background</h3>
+
+<p>
+  This sample illustrates the use of
+  <a href="https://w3c.github.io/presentation-api/">Presentation API</a>,
+  which gives the ability to access external presentation-type displays and use
+  them for presenting web content. The <code>PresentationRequest</code> object
+  is associated with a request to initiate or reconnect to a presentation made
+  by a controlling browsing context and it takes in a presentation request URL
+  when constructed. A presentation can be started by calling the
+  <code>start()</code> method on the <code>PresentationRequest</code> object.
+</p>
+
+{% capture initial_output_content %}
+<p>
+  <button id="newPresReq">new PresentationRequest()</button>
```

Comment:
nit: here and below, "Pres" can be removed from the ids. e.g. "createNewRequest", "start", "reconnect".